experiments/AMIP/moist_mpi_earth/coupler_driver.jl

```diff
@@ -120,19 +129,20 @@ end
 
 # init coupler
 coupler_field_names = (:T_S, :z0m_S, :z0b_S, :ρ_sfc, :q_sfc, :albedo, :F_A, :F_E, :F_R, :P_liq)
-coupler_fields =
+coupler_fileds =
     NamedTuple{coupler_field_names}(ntuple(i -> ClimaCore.Fields.zeros(boundary_space), length(coupler_field_names)))
 model_sims = (atm = atmos_sim, ice = ice_sim, lnd = land_sim, ocn = ocean_sim)
 dates = (; date = [date], date0 = [date0], date1 = [date1])
 
-coupler_sim = CouplerSimulation(
+cs = CouplerSimulation(
     FT(Δt_cpl),
     integrator.t,
+    tspan,
     dates,
     boundary_space,
     FT,
     mask,
-    coupler_fields,
+    coupler_fileds,
```

Comment:
same here

Reply:
Ahh, I missed this guy during the update! Thanks for catching it! 🐛